README.md

```diff
@@ -1,3 +1,5 @@
+![hugo-coder logotype a](https://user-images.githubusercontent.com/36028424/40582328-583f656c-61a5-11e8-8d86-6e7c23f2b45d.png)
```

Comment:
Please, fix this URL to something like this... following the screenshot example.
`https://github.com/luizdepra/hugo-coder/blob/master/images/hugo-coder-logotype-a.png`

 Also, you can remove the Hugo Coder title of the next line. Only showing the logo is enough.